ModularTegustation/tegu_items/representative/research/rcorp.dm

```diff
@@ -53,30 +53,68 @@
 	ItemUnlock(caller.order_list, "R Corp Rush Blade",	/obj/item/ego_weapon/city/rabbit_rush, 900)
 	ItemUnlock(caller.order_list, "R Corp Reindeer Staff",	/obj/item/ego_weapon/city/reindeer, 900)
 	..()
+//4th and 5th pack locks
+/datum/data/lc13research/fourthpack
+	research_name = "4th Pack"
+	research_desc = "We'll allow you the supply of some of our 4th pack rabbits, they're a little pricey though."
+	cost = AVERAGE_RESEARCH_PRICE
+	corp = R_CORP_REP
+
+/datum/data/lc13research/fifthpack
+	research_name = "5th Pack"
+	research_desc = "We'll give you some of our shoddy 5th pack, they're not as good as the 4th pack but they're cheap."
+	cost = LOW_RESEARCH_PRICE
+	corp = R_CORP_REP
 
 // ERTs
+
+/datum/data/lc13research/mobspawner/scoutraven
+	research_name = "4th Pack Scout Raven Team"
+	research_desc = "We can send in a couple of ravens to be 'spies' for you."
```

Comment:
Spies and not scouts?

Reply:
I'm going to be repurposing ravens soon to act more as eyes and ears for the rep in a future update.